### DISCUSSION TITLE
.NET troubleshooting.

### DISCUSSION THREAD

**Discussion by Sin-Shadow-Fox** [SteamRE/DepotDownloader#505] [Q&A]
https://github.com/dotnet/core/issues/7556#issuecomment-2161718205

**Comment by yaakov-h** [comment]
Whilst the content of third-party tutorials are beyond our scope, judging by your screenshots you veered wildly from the script in step 6 of the tutorial you were following.

DD is also published as a self-contained application so that tutorial is somewhat out of date.

**Reply by Sin-Shadow-Fox** [reply]
Step 6? I'm still on step 1.

>  Step 1 - Installing .NET Core
Go to Microsoft's Website [dotnet.microsoft.com] and make sure to download and install the suitable and latest version for your operating system.

**Reply by yaakov-h** [reply]
You can skip that these days, but DepotDownloader is only supported on platforms that can run .NET 8, which for Windows is Windows 10 v1607 and higher (including Windows 11).

**Reply by Sin-Shadow-Fox** [reply]
Thank you for opening my discussion. I've heard that it's possible to get .NET 8 to work on windows 7 but i don't understand how they did it > https://github.com/dotnet/core/issues/7556

perhaps you can make sense of it?

**Reply by yaakov-h** [reply]
There's no info in that Issue on doing so.

If .NET 8 / DepotDownloader works on Windows 7 then you're lucky, but if there are any problems whatsoever then neither Microsoft nor the SteamRE team will be able to fix it.

**Reply by Sin-Shadow-Fox** [reply]
> There's no info in that Issue on doing so.

Wait, whats going here then? Did i misinterpret what they're talking about?
https://github.com/dotnet/core/issues/7556#issuecomment-1786377918
